test/framework/easyblock.py

```diff
@@ -1025,10 +1025,16 @@ def test_init_extensions(self):
 
         test_ec = os.path.join(self.test_prefix, 'test.eb')
         test_ec_txt = toy_ec_txt.replace("('barbar', '0.0', {", "('barbar', '0.0', {'easyblock': 'DummyExtension',")
+        test_ec_txt = test_ec_txt.replace("('barbar', '0.0', {", "('barbar', '0.0', {'maxparallel': 3,")
```

Comment:
Isn't this changing the same as the line above? Maybe combine both into one? This looks like a C&P mistake